subworkflows/local/solid_biopsy/main.nf

```diff
@@ -88,6 +89,12 @@ workflow SOLID_BIOPSY {
                         ch_reports = ch_reports.mix(CIN_SIGNATURE_QUANTIFICATION.out.sig_activity_plot)
                 }
 
+                if (params.hrdcna_compute_score){
```

Comment:
As per the `task.when` mention earlier, ensure it runs only if compatible options are set.

Reply:
Let's scratch my suggestion, because upstream Nextflow is deprecating `when`.